tests/app/celery/test_research_mode_tasks.py

```diff
@@ -21,15 +22,19 @@
 )
 
 
-def test_make_sns_callback(notify_api, rmock):
+@freeze_time("2018-01-25 14:00:30")
+def test_make_sns_callback(notify_api, mocker):
```

Comment:
Can you write additional tests (or use pytest params) when using magic numbers for tmp fail/permanent failure/delivered?